src/containers/access/src/sagas/updateAccessRequest.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import isPlainObject from 'lodash/isPlainObject';
+import {
+  call,
+  put,
+  select,
+  takeEvery,
+} from '@redux-saga/core/effects';
+import { Types } from 'lattice';
+import {
+  DataApiActions,
+  DataApiSagas,
+} from 'lattice-sagas';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { AppTypes, PropertyTypes } from '../../../../core/edm/constants';
+import { selectPropertyTypeIDsByFQN } from '../../../../core/redux/selectors';
+import { getESIDFromConfig } from '../../../../utils/AppUtils';
+import { ERR_ACTION_VALUE_TYPE } from '../../../../utils/Errors';
+import { APP_PATHS } from '../../../app';
+import {
+  UPDATE_ACCESS_REQUEST,
+  updateAccessRequest,
+} from '../actions';
+
+const { updateEntityData } = DataApiActions;
+const { updateEntityDataWorker } = DataApiSagas;
+const { UpdateTypes } = Types;
+
+const { ACCESS_REQUEST_SUBMISSION } = AppTypes;
+const { FORM_DATA } = PropertyTypes;
+
+const LOG = new Logger('updateAccessRequest');
+
+function* updateAccessRequestWorker(action :SequenceAction) :Saga<void> {
+  const response :Object = {};
+  try {
+    const { value } = action;
+    if (!isPlainObject(value)) throw ERR_ACTION_VALUE_TYPE;
+    const { formData, entityKeyId } = value;
+    const formDataStr = JSON.stringify(formData);
+    yield put(updateAccessRequest.request(action.id, {
+      path: ['accessRequest', FORM_DATA.toString(), 0],
+      formData: formDataStr
+    }));
```

Comment:
Just curious, why are you doing this?

Reply:
So the form data doesn't revert itself to it's unupdated state when a user submits changes